moveit_setup_assistant/moveit_setup_core_plugins/src/start_screen.cpp

```diff
@@ -48,16 +48,7 @@ void StartScreen::onInit()
 void StartScreen::loadURDFFile(const std::string& urdf_file_path, const std::string& xacro_args)
 {
   urdf_config_->loadFromPath(urdf_file_path, xacro_args);
-}
-
-void StartScreen::setSRDFFile(const std::string& srdf_string)
-{
-  srdf_config_->loadSRDFString(srdf_string);
-}
-
-std::string StartScreen::getURDFName()
-{
-  return urdf_config_->getName();
+  srdf_config_->updateRobotModel();
```

Comment:
what does this do? At this point, there is not really a RobotModel yet, right?

Reply:
It [creates the shared RobotModel object](https://github.com/ros-planning/moveit2/blob/93be6385cdb294a7b82470c29fb98dde93e319a3/moveit_setup_assistant/moveit_setup_framework/src/srdf_config.cpp#L147), which is basically empty, but once the URDF is loaded, the robot name can be set. It's equivalent to the removed loading of the blank SRDF file.